sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/attestation/attestation_client_models.hpp"
+#include "azure/attestation/attestation_client_options.hpp"
+#include <azure/core/context.hpp>
+#include <azure/core/url.hpp>
+#include <string>
+
+namespace Azure { namespace Core { namespace Http { namespace _internal {
+  class HttpPipeline;
+}}}} // namespace Azure::Core::Http::_internal
+
+namespace Azure { namespace Security { namespace Attestation {
+
+  /**
+   *
+   * The AttestationAdministrationClient implements the functionality required by the
+   * "Administration" family of attestation service APIs.
+   *
+   * @note: Attestation administration APIs cannot be used on shared attestation service instances.
```

Comment:
Where do we imagine this doc comment to be visible for the customer? Would it render in the generated doxygen as we expected on the `AttestationAdministrationClient` class? Should we split it up in a `@brief` and other annotations?

Reply:
I hope that documentation comments appear in the documentation. Or at least in intellisense.